susemanager-utils/susemanager-sls/susemanager-sls.changes.mczernek.context_manage_fix_reposync

```diff
@@ -0,0 +1 @@
+- Fix python sonarlint issue S5754
```

Comment:
I'd drop this, users are not interested in this change and they never saw it in the first place (since the code was introduced just a day ago)

Reply:
I needed to add this changelog; otherwise, the chlog check was failing that we're changing a file without a chlog entry in a package. Is there a way to disable the check?